p2panda-auth/Cargo.toml

```diff
@@ -3,20 +3,30 @@ name = "p2panda-auth"
 version = "0.1.0"
 edition = "2024"
 authors = [
-  "adz <x12@adz.garden>",
-  "sandreae <contact@samandreae.com>",
-  "glyph <glyph@mycelial.technology>",
+    "adz <x12@adz.garden>",
+    "sandreae <contact@samandreae.com>",
+    "glyph <glyph@mycelial.technology>",
 ]
 repository = "https://github.com/p2panda/p2panda"
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 keywords = ["auth", "access-control", "groups"]
 
+[features]
+default = []
+test-utils = ["rand"]
```

Comment:
Minor thing for consistency, but we use `test_utils` (with underscore) across all crates so far